Misc/NEWS.d/next/Core and Builtins/2022-05-20-13-32-24.gh-issue-15870.e9B-pv.rst

```diff
@@ -0,0 +1,13 @@
+Added the new function :c:func:`PyType_FromMetaModuleAndSpec`, which mirrors
+the behavior of :c:func:`PyType_FromModuleAndSpec`, except that it takes an
+additional metaclass argument. This is useful for language binding tools, where
+it enables use cases including
+
+- storing additional binding-related data structures in the type object.
+
+- intercepting type-related operations like subclassing or static attribute
+  access by specifying a metaclass with custom slots.
+
+Importantly, :c:func:`PyType_FromMetaModuleAndSpec` is available in the Limited
+API, which provides a path towards migrating more binding tools onto the Stable
+API.
```

Comment:
Limited API, Stable ABI :)

```suggestion
ABI.
```

Reply:
Also, the docs test fail because your news entry is numbered incorrectly (15870 instead of 93012). Either `git mv` it so it is named correctly, or `git rm` it, and add a new blurb entry using the correct GitHub Issues number.

Reply:
Sorry about tagging your username in the commits, I will squash them and remove those messages.

Reply:
Please just leave them for now. Rewriting history makes reviewing harder, and that is more important for this PR. Just keep it in mind for future PRs :)